android/app/src/main/java/rx/Observable.java

```diff
@@ -0,0 +1,6 @@
+package rx;
+
+public class Observable {
+    // Dummy class required for Jackson-Databind support if
+    // RxJava is not a project dependency.
```

Comment:
Yes it is but  For realm we have not used Observable. 
https://realm.io/docs/java/latest/#rxjava